README.md

```diff
@@ -1,12 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-Profile Moblie Verison
+**Profile Moblie Verison**
 
 ### Description of the project.
 
 The Html & Css Mobile View Project is a mimic of figma web templates  
 
+**Portifolio project for mobile view**
+
 
 ## Built With
 
```

Comment:
- [OPTIONAL] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd project_name`, etc 🙃  Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.